java/test/jmri/jmrit/logixng/tools/ImportEntryExitTest.java

```diff
@@ -49,20 +49,25 @@ private void deleteAllLogixs() {
         }
     }
 
-    private boolean isEnabled(DestinationPoints dp) throws NoSuchMethodException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+    private boolean isEnabled(DestinationPoints dp) {
 //        Method retrieveItems = dp.getClass().getDeclaredMethod("isEnabled", String.class);
-        Method isEnabled = dp.getClass().getDeclaredMethod("isEnabled");
-        isEnabled.setAccessible(true);
-        return (boolean)isEnabled.invoke(dp);
+        try {
+            Method isEnabled = dp.getClass().getDeclaredMethod("isEnabled");
+            isEnabled.setAccessible(true);
+            return (boolean)isEnabled.invoke(dp);
+        }
+        catch ( IllegalAccessException | IllegalArgumentException | NoSuchMethodException | SecurityException | InvocationTargetException ex ){
+            return false;
```

Comment:
Please add an error log statement in these cases. Otherwise it's difficult to know why it went wrong.

`log.error("Exception thrown", ex);`

But it's probably be better to change the `isEnabled()` method to public and add a comment that it's used by tests.

Reply:
Seems a lot cleaner to make the method public :-)